src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1604,6 +1604,123 @@ void valid() {
             }
         }
 
+        @Nested
+        class InformativeJustification {
+
+            @Test
+            void invalidArgumentInformativeJustification() {
```

Comment:
The only thing I'd like to note is that this doesn't fail because of the recent changes (e.g., because of `aida:InformativeJustificationMembersUniqueParentDoc`).  It fails because `aida:RelationArgumentShape` is a closed shape.  If the the shape is opened (via editing aida_ontology.shacl) it passes validation.

Maybe that's perfectly okay...I just thought I'd point it out.

Reply:
This is a fair point. I'll remove this test.

Reply:
Fixed.